### PR TITLE
Update cms-eventdisplay-files-Run2011A.xml

### PR DESCRIPTION
**Pull request by AnxhelaDani**
I linked the event display files to the correct primary datasets

### CODE REVIEW

**Comment by pherterich** [comment]
Will close #843

File: invenio_opendata/testsuite/data/cms/cms-eventdisplay-files-Run2011A.xml [hunk 1]
```diff
@@ -47,9 +47,9 @@
       <subfield code="e">CMS</subfield>
     </datafield>
     <datafield tag="772" ind1=" " ind2=" ">
-      <subfield code="a">FT_53_LV5_AN1</subfield>
-      <subfield code="o"></subfield>
-      <subfield code="w">1</subfield>
+      <subfield code="a">/BTag/Run2011A-12Oct2013-v1/AOD</subfield>
+      <subfield code="o">10.7483/OPENDATA.CMS.N372.QF6S</subfield>
+      <subfield code="w">15</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-eventdisplay-files-Run2011A.xml]: `772__o` is a new subfield.  I can add it to the data model.  Do you plan to use it more widely?  Do you plan to store other PIDs there?  (if yes, then we may want to consider indicating the PID type...)

Comment (pherterich) [inline invenio_opendata/testsuite/data/cms/cms-eventdisplay-files-Run2011A.xml]: That's weird, it's already in the records of last year's event display files...
I would only use it for DOIs and cover other cases with $u and the URL

Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-eventdisplay-files-Run2011A.xml]: OK, added in 0ea0a3fad6d40cafe0e6168e3fdeec997b4fe052.

**Comment by tiborsimko** [comment]
Thanks, merged as c6fc5b1e23878122fd5d8f2fa7f53a46a6def9bf

**Pull request closed without merging**